### ISSUE TITLE
Feature Request: Delete collection after downloading is finished

### ISSUE THREAD

**Issue by moreRGB** [Richard-Weiss/Bing-Creator-Image-Downloader#43]
Hi Richard,

thanks for the excellent tool. Adding the prompt to the EXIF metadata is brilliant!

I would like to ask for a feature: Delete the collection after downloading all the images is finished.

Currently one still needs to visit https://www.bing.com/saves to delete the collection (to prevent reaching the 1000 images limit and/or to not download the same images again).

It seems that this is just an single HTTP-API call.

**Comment by Richard-Weiss** [comment]
@moreRGB 
I've looked into something similar when trying to download the images in batches of 1000 and deleting them after that.  
Sadly the API has a very low throughput and I encountered errors when having too many requests in parallel.

However, I looked into some alternatives and there's also an API endpoint for creating new collections and for moving images to collections.
Maybe it would be more feasible to have an option to move the images to a newly created collection after downloading them.
It's a single request instead of multiple which should speed up the process by a lot.

**Comment by Richard-Weiss** [comment]
@moreRGB
I've created a new issue for moving the images to new collections, as it's a separate feature.
I'll prioritize this issue, as it's smaller in scope.

**Comment by Richard-Weiss** [comment]
@moreRGB 
I haven't messed with the collection item move API much more, but it seemed a lot simpler than the collection item add API.
There's also a copy endpoint, which is the same as the move one except for the url but retains the images in the original collection.
The collection item delete API is already working quite well in Insomnia from what I tested.
I'll add some guardrails and additional confirmation in the settings for the collection deletion.
